modules/programs/ssh.nix

```diff
@@ -476,6 +492,10 @@ in
       ${concatStringsSep "\n" (
         mapAttrsToList (n: v: "${n} ${v}") cfg.extraOptionOverrides)}
 
+      ${optionalString (cfg.includes != [ ]) ''
```

Comment:
It would be nice to combine this with the `concatStringsSep` blocks above and below this into a single `concatStringsSep`:

```nix
${concatStringsSep "\n" (
  (mapAttrsToList (...))
  ++ (optional (cfg.includes != [ ]) "..."
  ++ (map (block: ...) ...))}
```